### PR TITLE
feat: Change building image from Docker  to s2i

### LINKED ISSUE #30: Convert image to s2i ubi8 based node16 image
Use ubi8 based node16 base image and `build-strategy: Source` in `.aicoe-ci.yaml` Address that in the dev config. #22 

https://github.com/AICoE/aicoe-ci supports a `Source` strategy. Ideally we would like to ditch `Dockerfile` in this repo and build using https://github.com/openshift/source-to-image from an appropriate stable (as newest version as possible, even versions are stable) NodeJS on UBI8 base image from https://catalog.redhat.com/software/containers/search

### PR DESCRIPTION
**Pull request by mimotej**
Resolves #30 
Added a script which will create an image called `peribolos-as-a-service`. Removed unused docker files and updated `README.md`.

### CODE REVIEW

**Comment by harshad16** [comment]
i understand this is still WIP.
just one comment:
`registry.redhat.io/ubi8/nodejs-16:latest` would require redhat subscribed creds to build.
can we change it with `registry.access.redhat.com/ubi8/nodejs-16:latest`, it is available.

**Comment by mimotej** [comment]
> i understand this is still WIP. just one comment: `registry.redhat.io/ubi8/nodejs-16:latest` would require redhat subscribed creds to build. can we change it with `registry.access.redhat.com/ubi8/nodejs-16:latest`, it is available.

@harshad16 good point thanks :+1: .

File: scripts/build-image.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Base image
+NODEJS_BASE_IMAGE=nodejs-16
```
Comment (tumido) [inline scripts/build-image.sh:4]: Can we source the base image from `.aicoe-ci.yaml` instead?

**Review by tumido — COMMENTED** [review]
I like it! Good content. I have a single little nit that we may want to adjust, WDYT @mimotej ?

File: scripts/build-image.sh (hunk 1 above)
Comment (mimotej) [reply scripts/build-image.sh]: Good Idea, changed it WDYT? @tumido

Comment (tumido) [reply scripts/build-image.sh]: I :heart: it @mimotej  Looks great!

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by harshad16** [comment]
I think I m late to the party, just one question for my self-knowledge, 
why do we have a separate script, the description of using the `s2i build` command would have done the trick, right?
PS: no changes needed, just asking for knowledge

**Pull request merged**